crates/core/tedge_api/src/messages.rs

```diff
@@ -60,6 +61,10 @@ impl SoftwareListRequest {
     pub fn topic_name() -> &'static str {
         "tedge/commands/req/software/list"
```

Comment:
How about making this "tedge/commands/req/software/list" as a const and getting rid of this function?